pkgs/by-name/ni/nicstat/package.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+}:
+
+stdenv.mkDerivation rec {
+  pname = "nicstat";
+  version = "0-unstable-2018-05-09";
+
+  src = fetchFromGitHub {
+    owner = "scotte";
+    repo = "nicstat";
+    rev = "1fbe28198b49a2062b0c928554f93db33cb288c3";
+    hash = "sha256-7+11K9636dGeW0HaaH6OJF5Wy4CXYXfoaZOVfhHK6kg=";
+  };
+
+  buildPhase = ''
+    runHook preBuild
+
+    $CC -O2 nicstat.c -o nicstat
+
+    runHook postBuild
+  '';
+
+  installPhase = ''
+    runHook preInstall
+
+    install -d "$out/bin"
+    install -m 755 nicstat "$out/bin"
+
+    install -d "$out/share/man/man1"
+    install -m 644 nicstat.1 "$out/share/man/man1"
+
+    runHook postInstall
+  '';
+
+  meta = with lib; {
```

Comment:
```suggestion
  meta = {
```

Reply:
Isn't the `with lib;` usage generally accepted for the `meta` attribute set?

Reply:
Ehr... nope. The word is _tolerated_, not _accepted_.

A not so short time ago we argued a lot about constraining the usage of `with` in Nixpkgs.
It was generally accepted a `with` embracing the meta set.

However, since the Pacman Version Incident, its acceptance is not clear anymore.

https://github.com/NixOS/nixpkgs/issues/292468#issuecomment-2016454242

I am more strict, and in order to avoid footguns, I push for the removal of `with` in scopes larger than one line.